python_on_whales/components/network/cli_wrapper.py

```diff
@@ -209,6 +210,26 @@ def disconnect(
         full_cmd += [network, container]
         run(full_cmd)
 
+    def exists(
+        self,
+        network: str,
+    ) -> bool:
+        """Check if a network exists
+
+        Although there is an actual exists command for Podman,
+        it is instead implemented using inspect for compatibility
+        with Docker, which does not support it as a standalone command.
```

Comment:
I think this belongs as a comment rather than a docstring - the caller shouldn't care how it's implemented

Reply:
Seems like this same comment applies to other objects with an `exists()` method, and those don't have any similar info on the documentation, so I'll just remove it.